### PR TITLE
Jordan/notifications fix

### PR DESCRIPTION
**Pull request by jbibla**
Closes #ISSUE

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: app/src/router.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ export const routeGuard = (store) => async (to, from, next) => {
     if (store.state.account.userSignedIn) {
       next()
     } else {
-      next(`/paywall`)
+      next(`/notification-wall`)
```
Comment (Bitcoinera) [inline app/src/router.js:24]: ?? :eyes:

Comment (Bitcoinera) [reply app/src/router.js:24]: You would need to change the routes thing too no?

Comment (Bitcoinera) [reply app/src/router.js:24]: ah it works :shrug: Magic

Comment (jbibla) [reply app/src/router.js:24]: i did in a previous PR - but forgot this one link that still referenced `/paywall`

Comment (Bitcoinera) [reply app/src/router.js:24]: shouldn't it be `notifications-wall` though? In plural

**Pull request merged**